common-files/utils/gas.mjs

```diff
@@ -37,15 +37,15 @@ export async function estimateGas(tx, web3, gasDefault = 4000000, gasMultiplier
  *
  * @async
  * @function estimateGasPrice
- * @param {string} gasEstimateEndpoint URL for querying gas prices
  * @param {Web3} web3
- * @param {number} gasPriceDefault Has default
- * @param {number} gasPriceMultiplier Buffer to apply to estimated gas price - Has default
+ * @param {string} [gasEstimateEndpoint] URL for querying gas prices - Has default
+ * @param {number} [gasPriceDefault] Has default
+ * @param {number} [gasPriceMultiplier] Buffer to apply to estimated gas price - Has default
  * @returns {Promise<number>}
  */
 export async function estimateGasPrice(
-  gasEstimateEndpoint,
   web3,
+  gasEstimateEndpoint = 'https://vqxy02tr5e.execute-api.us-east-2.amazonaws.com/production/estimateGas',
```

Comment:
Where is this endpoint located?

Reply:
GM @druiz0992 the value is the same as per the config, but since common files is a library, I prefer to avoid using the config. Or do you think we will stop this service? We could pass also empty string and will default to the `web3` option - this function has 2 backup options.

Reply:
This is a good point, as we will deploy this somewhere the service should be consumed from this "new" stack. We should create a new environment var for this endpoint.

CC: @imagobea @druiz0992